test/e2e/multihoming.go

```diff
@@ -452,7 +452,51 @@ var _ = Describe("Multi Homing", feature.MultiHoming, func() {
 					containerCmd: httpServerContainerCmd(port),
 					hostNetwork:  true,
 				},
-				true, // collocated on same node
+				true, // collocated on the same node
+				Label("STORY", "SDN-5345"),
+			),
+			Entry(
+				"can be reached by a host-networked pod on a different node",
```

Comment:
nit: this name and the ones above good to clarify host-neteworked pod on a different node to local net pod v/s localnet->host is the can reach is it?

so can be reached is host-networked pod to localnet pod
and can reach is localnet pod to host-networked pod

Reply:
I had to play with the verb (`can be reached` vs `can reach`) in order to be under the same outer `Context("A single pod with an OVN-K secondary network", func() {`. 
I'll add a comment to clarify what we're testing